doc/examples/data/plot_specific.py

```diff
@@ -38,7 +38,7 @@
 fig, axes = plt.subplots(1, 2, figsize=(8, 4))
```

Comment:
```suggestion
fig, axes = plt.subplots(ncols=2, figsize=(8, 4))
```
It's just more consistent wrt the other gallery examples.